pyresample/test/test_geometry.py

```diff
@@ -784,40 +784,40 @@ def test_lonlat2colrow(self):
     def test_colrow2lonlat(self):
         """Test colrow2lonlat."""
         from pyresample import utils
-        area_id = 'meteosat_0deg'
-        area_name = 'Meteosat 0 degree Service'
-        proj_id = 'geos0'
-        x_size = 3712
-        y_size = 3712
-        area_extent = [-5570248.477339261, -5567248.074173444,
-                       5567248.074173444, 5570248.477339261]
-        proj_dict = {'a': '6378169.00',
-                     'b': '6356583.80',
-                     'h': '35785831.0',
-                     'lon_0': '0.0',
-                     'proj': 'geos'}
```

Comment:
Why remove this test case ? I mean the one you added makes perfect sense, but it doesn't hurt with an extra usecase.

Reply:
Good point, thanks! I reintroduced the test.